gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/resources/portal-openapi.yaml

```diff
@@ -3755,6 +3816,13 @@ components:
                     $ref: "#/components/schemas/MetadataMap"
                 links:
                     $ref: "#/components/schemas/Links"
+        MessageLogsResponse:
+          properties:
+            data:
+              description: Page of messages
+              type: array
+              items:
+                $ref: "#/components/schemas/MessageLog"
```

Comment:
I may be wrong but when I have read the `application-log.service.ts` it returns an `AggregatedMessageLogsResponse` which is not only composed of the data. So is it up to date. Should we add the links and metadata here or am I missing something  ?